tests/index.test.js

```diff
@@ -56,4 +56,38 @@ describe('Asset', () => {
     },
     25000
   )
+
+  it(
+    'should work with multiple entry points',
+    async () => {
+      // Init bundler
+      const bundler = new Bundler(
+        [path.join(__dirname, './file.js'), path.join(__dirname, './file2.js')],
```

Comment:
Hello! Thank you very much for the PR!
I have issues running this new test: I think file2.js is missing.

Reply:
yeah, forgot to add it, but now it's fixed and I've also rebased it to be up-to-date with master branch.

Btw, thanks for the plugin :)